src/gondi.ts

```diff
@@ -413,6 +413,14 @@ export class Gondi {
     });
   }
 
+  async withdrawalPositions({ limit = 20, cursor, ...rest }: model.ListWithdrawalPositionsProps) {
```

Comment:
Let's bump the version but also add `  /** @internal */` to all new methods so that are not included in the docs